docs/topics/db/aggregation.txt

```diff
@@ -643,6 +643,55 @@ aggregate that author count, referencing the annotation field:
     >>> Book.objects.annotate(num_authors=Count("authors")).aggregate(Avg("num_authors"))
     {'num_authors__avg': 1.66}
 
+When you are generating an aggregate by using ``.values()`` clause to create the grouping
+for the annotation, it may be necessary to apply :class:`~django.db.models.AnyValue`
+when the annotation includes a mix of aggregated and non-aggregated functions. This
+happens when the database is not be able to determine the functional dependency of the
+columns in the non-aggregated functions to the grouping column.
+
+Take the following example:
+
+.. code-block:: pycon
+
+    >>> from django.db.models import F, Count, Greatest
+    >>> Book.objects.annotate(greatest_pages=Greatest("pages", 600)).values(
+    ...     "greatest_pages"
+    ... ).annotate(
+    ...     num_authors=Count("authors"),
+    ...     pages_per_author=F("greatest_pages") / F("num_authors"),
+    ... ).aggregate(
+    ...     Avg("pages_per_author")
+    ... )
+
+This creates groups of books based on the SQL column ``GREATEST(pages, 600)``.
+One unique group consists of books with 600 pages or less, and other unique
+groups will consist of books with the same pages. The ``pages_per_author``
+involves a mix of aggregated and non-aggregated functions, because
+``greatest_page`` is a non-aggregated function, whereas the ``num_authors`` is
+an aggregated function.
+
+The database is not able to determine that the ``greatest_pages`` used in
+``pages_per_author`` is functionally dependent on the column in the group by
+clause. In MySQL, this will happen if the ``sql_mode`` contains
+``ONLY_FULL_GROUP_BY``, and it will raise the
+``OperationalError: incompatible with sql_mode=only_full_group_by`` error.
```

Comment:
Reading this as someone who doesn't know much about the feature, do I need to be worried about this error with non-mysql databases? If the answer is no, do we still want to mention the use case of why we should use `AnyValue` with non-mysql databases?

Reply:
> Reading this as someone who doesn't know much about the feature, do I need to be worried about this error with non-mysql databases?

I think we don't and share your sentiment that the documentation should be adjusted accordingly.